app/src/main/res/layout/fragment_experiences.xml

```diff
@@ -7,21 +7,26 @@
     tools:context=".fragments.ExperiencesFragment">
 
     <TextView
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
         android:id="@+id/completed_experiences_user_points"
-        android:textAppearance="?attr/textAppearanceSubtitle1"
-        android:layout_marginEnd="64dp"
-        android:layout_marginStart="16dp"
-        android:layout_marginVertical="32dp"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="35dp"
+        android:layout_marginTop="90dp"
+        android:text="@string/experience_completed_title"
+        android:textAppearance="@style/TextAppearance.Material3.HeadlineLarge"
+        android:textColor="?attr/colorOnSurface"
+        app:fontFamily="@font/pt_sans_bold"
+        app:layout_constraintBottom_toTopOf="@+id/completed_experiences_recycler"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintBottom_toTopOf="@+id/completed_experiences_recycler"/>
+        app:layout_constraintHorizontal_bias="0.0"
```

Comment:
A bias of `0.0` is the same as removing one of the two horizontal constraints

Reply:
It works only if I remove the `app:layout_constraintEnd_toEndOf="parent"` constraint.

Reply:
Yup that's right, same as in `fragment_user_profile.xml`